.travis.yml

```diff
@@ -1,11 +1,42 @@
-language: minimal
+language: generic
+
+matrix:
+  include:
+    - os: linux
+      env:
+        - ARCH=ppc64le
+
+    - os: linux
+      env:
+        - ARCH=aarch64
+
+    - os: linux
+      env:
+        - ARCH=amd64
+
+before_install:
+  # Enable experimental features (needed for squash)
+  - sudo cp docker_daemon_config.json /etc/docker/daemon.json
+  - sudo service docker restart
+
+  - docker info
+  - docker run --rm --privileged multiarch/qemu-user-static:register --reset
 
 install:
-  - ./build.sh
+  - docker build -t docker_constructor -f Dockerfile.${ARCH} .
+
+script:
+  - echo "* Fix permissions"
+  - chmod 777 build/
+  - chmod 666 Miniforge3/construct.yaml
+  - echo "* Build Miniforge installer"
+  - docker run --rm -ti --mount type=bind,source="$(pwd)",target=/construct docker_constructor
+  - echo "* Test Miniforge installer"
+  - docker run --rm -ti --mount type=bind,source="$(pwd)",target=/construct forge_constructor /construct/test.sh
```

Comment:
We should test this on a docker image without conda already installed.

Reply:
What image would you suggest? I thought about it but it keeps the repo simply because I don't have to add new `Dockerfile`. Maybe I can delete conda from `test.sh`?

Reply:
I don't think we can delete the conda installation there, because there are some packages like tini used there. We can use the docker images from https://github.com/conda-forge/docker-images and provide a way there to use a local installer instead of https://github.com/conda-forge/docker-images/blob/master/scripts/run_commands

Reply:
Also in `test.sh`, I execute `which python` so we know what Python we are executing. Maybe it's enough.

Reply:
That wouldn't test that the installer can be installed on a system without conda, but only that a new python is installed.

Reply:
See https://travis-ci.com/conda-forge/miniforge/jobs/256852329#L839

Reply:
Ok, I see your point know. What do you think about merging this and we can address this in another PR?

Reply:
Before https://github.com/conda-forge/docker-images/blob/eb117c900c58263843d6e5f89eb5b72ddbc92d45/scripts/run_commands#L7 we could check for an installer file in a defined mounted directory and use it if we detect it? I'll open a ticket there.

Reply:
See https://github.com/conda-forge/docker-images/issues/122